hazelcast/src/main/java/com/hazelcast/jet/impl/execution/TaskletExecutionService.java

```diff
@@ -85,6 +87,7 @@
     private final String hzInstanceName;
     private final ILogger logger;
     private int cooperativeThreadIndex;
+    private final AtomicBoolean startCooperativeThreads = new AtomicBoolean(false);
```

Comment:
We no longer need this - we can start the threads right away and they won't take cpu.